bin/server/cmd/main_omni.go

```diff
@@ -0,0 +1,61 @@
+//go:build omni
```

Comment:
Lots of main code can be shared, I wonder if we could just have two files postgres_embedded.go and postgres_omni/slim.go with exact functions, however, some of the functions in postgres_slim.go are no-op such as installing/starting postgres binary.

Reply:
I would generalize it a bit like

component.go
component_slim.go

We may add some other stuff later (e.g. mail server)

Reply:
Thanks for your feedbacks. Let me give it a try.